WcaOnRails/spec/requests/oauth/oauth_spec.rb

```diff
@@ -18,7 +24,7 @@
   end
 
   context "grant_type authorization" do
-    let(:oauth_app) { FactoryBot.create(:oauth_application, redirect_uri: "urn:ietf:wg:oauth:2.0:oob\n#{oauth_authorization_url}") }
```

Comment:
Here we've been relaying on the fact that in multi-line `redirect_uri` all URIs after `urn:ietf:wg:oauth:2.0:oob` were not validated, so it was fine for them to be just `http`. It's been a bug and was fixed in https://github.com/doorkeeper-gem/doorkeeper/pull/976, so I added some code above pretending we use SSL in those tests.